acme/acme/client.py

```diff
@@ -509,15 +518,19 @@ class ClientNetwork(object):  # pylint: disable=too-many-instance-attributes
     JSON_ERROR_CONTENT_TYPE = 'application/problem+json'
     REPLAY_NONCE_HEADER = 'Replay-Nonce'
 
-    def __init__(self, key, alg=jose.RS256, verify_ssl=True,
-                 user_agent='acme-python', timeout=DEFAULT_NETWORK_TIMEOUT):
+    # pylint: disable=too-many-arguments
+    def __init__(self, key, account=None, alg=jose.RS256, verify_ssl=True,
+                 user_agent='acme-python', timeout=DEFAULT_NETWORK_TIMEOUT,
+                 acme_version=2):
```

Comment:
Why make the default here 2 rather than 1? Why not keep existing behavior unless you ask for something different?

Reply:
Whoops, this was a mistake, accidentally left in. Fixed now.